.travis.yml

```diff
@@ -311,31 +294,448 @@ jobs:
       env: DB=agnostic
       install:
         - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - composer install --ignore-platform-reqs # PHPUnit doesn't accept PHP8
+        - COMPOSER=./tests/composer/composer-php8.json composer install
         - composer show
     - php: nightly
       env: DB=mysql
       install:
         - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - composer install --ignore-platform-reqs
+        - COMPOSER=./tests/composer/composer-php8.json composer install
         - composer show
     - php: nightly
       env: DB=sqlite
       install:
         - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - composer install --ignore-platform-reqs
+        - COMPOSER=./tests/composer/composer-php8.json composer install
+        - composer show
+
+    - name: "PHP 7.4, Agnostic, Symfony ~2.7"
```

Comment:
I really needed all of these and more. You can read the commit history and run them individually if you doubt it.